src/Runner.Common/JobServerQueue.cs

```diff
@@ -71,14 +82,21 @@ public sealed class JobServerQueue : RunnerService, IJobServerQueue
 
         // Web console dequeue will start with process queue every 250ms for the first 60*4 times (~60 seconds).
         // Then the dequeue will happen every 500ms.
-        // In this way, customer still can get instance live console output on job start, 
+        // In this way, customer still can get instance live console output on job start,
         // at the same time we can cut the load to server after the build run for more than 60s
         private int _webConsoleLineAggressiveDequeueCount = 0;
         private const int _webConsoleLineAggressiveDequeueLimit = 4 * 60;
         private const int _webConsoleLineQueueSizeLimit = 1024;
         private bool _webConsoleLineAggressiveDequeue = true;
         private bool _firstConsoleOutputs = true;
 
+        private int totalBatchedLinesAttemptedByWebsocket = 0;
```

Comment:
>for long jobs one instance of failures will have us suddenly using the old (slower) method for all calls until that job ends.

we have 50% failure rate to consider here (with 10 minimum calls), I think in the end, we try our best to do websocket, which should be true in most cases, but it's not the end of the world if we couldn't for few jobs as blips happen, as we fallback to what we were doing before (so any concerns with this being a DDOS already exists today, but even if this happens where we can't use websocket, we are limiting the affect since this is scoped to few jobs).